index.html

```diff
@@ -200,17 +202,39 @@ <h3>New Normative Specifications Continuously Under Development</h3>
       associated <a href="https://www.w3.org/mid/cfe-6964-e72ae545706a44a8b42f44f11fc0b2e98a4feca3@w3.org">Call for Exclusion</a> on 2019-04-04 ended on 2019-06-03</p>
     </dd>
 
-    <dt id="imsc-hrm" class="spec">
-      <a href="https://www.w3.org/TR/imsc-hrm/">IMSC Hypothetical Render Model</a>
+    <dt id="imsc-1-3" class="spec">
+      <a href="https://www.w3.org/TR/ttml-imsc1.3/">TTML Profiles for Internet Media Subtitles and Captions 1.3</a>
     </dt>
     <dd>
-      <p>This specification specifies an Hypothetical Render Model (HRM) that constrains 
-        the complexity of an <a href="https://www.w3.org/TR/imsc-hrm/#dfn-imsc-document-instance">IMSC Document Instance</a>.</p>
-      <p class="draft-status"><b>Draft state:</b> <a href="https://www.w3.org/TR/imsc-hrm/">Working Draft</a></p>
+      <p>This specification defines two profiles of <a href="https://www.w3.org/TR/ttml2/">[ttml2]</a>:
+        a text-only profile and an image-only profile.
+        These profiles are intended to be used across subtitle and caption delivery applications worldwide,
+        thereby simplifying interoperability, consistent rendering and conversion to other subtitling and captioning formats.</p>
+      <p>This specification improves on <a href="https://www.w3.org/TR/ttml-imsc1.2/">[ttml-imsc1.2]</a> by supporting additional contemporary practices,
+        while retaining compatibility with <a href="https://www.w3.org/TR/ttml-imsc1.2/">[ttml-imsc1.2]</a> documents.</p>
+      <p class="draft-status"><b>Draft state:</b> <a href="https://www.w3.org/TR/ttml-imsc1.3/">Working Draft</a></p>
       <p><b>Adopted Working Draft:</b> <a href="https://www.w3.org/TR/2022/WD-imsc-hrm-20220106/">IMSC Hypothetical Render Model</a> (6 January 2022)</p>
```

Comment:
there is no adopted working draft, afaiu.

Reply:
done in 7015b33 if I understood your comment correctly @himorin